htmlproject.html

```diff
@@ -0,0 +1,100 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Html Project</title>
+    <link rel="stylesheet" href="htmlprojectstyles.css">
+</head>
+
+<body>
+    <header>
+        <div class="navBar">
+            <div>
+                <h2>ami<span>.</span>ke</h2>
+            </div>
+            <div>
+                <nav class="navItems">
+                    <a href="#home"> Home</a>
+                    <a href="#services"> Serivices</a>
+                    <a href="#about"> About</a>
+                    <a href="#skills"> Skills</a>
+                    <a href="#portfolio"> Portfolio</a>
+                    <a href="#blog"> Blog</a>
+                    <a href="#conact"> Contact</a>
+                </nav>
+
+            </div>
+        </div>
+    </header>
+    <div>
+        <img src="211.jpeg" alt="Head">
+    </div>
+
+    <main>
+
+        <div class="MI5a">
+            <h3>SERVICES</h3>
+            <h2>My services</h2>
+            <p style="width: 500px; text-align: center;">Lorem ipsum is simply a dummy text of the printing and
```

Comment:
I know this is not a CSS checkpoint, but let's try to not use inline CSS as much as possible as it's harder to maintain and scale.